src/server/index.jsx

```diff
@@ -73,7 +73,12 @@ server
     }),
   )
   .use(compression())
-  .use(helmet({ frameguard: { action: 'deny' } }))
+  .use(
+    helmet({
+      frameguard: { action: 'deny' },
+      contentSecurityPolicy: false,
```

Comment:
This is in place as we do not use helmet for our CSP right? that is a separate express middleware

Reply:
@chris-hinds yes, the default option for `contentSecurityPolicy` was `false` in the previous version